
Setting up data pipeline
Restric
Virtual devices cannot be modified after being initialized
Epoch 1/2
2022-05-28 01:19:34.365013: I tensorflow/stream_executor/cuda/cuda_dnn.cc:366] Loaded cuDNN version 8400






     13/Unknown - 24s 1s/step - loss: 0.1316 - accuracy: 0.8900
Traceback (most recent call last):
  File "main.py", line 58, in <module>
  File "/home/melih/anaconda3/envs/tf-web/lib/python3.8/site-packages/keras/utils/traceback_utils.py", line 67, in error_handler
    raise e.with_traceback(filtered_tb) from None
  File "/home/melih/anaconda3/envs/tf-web/lib/python3.8/site-packages/keras/callbacks.py", line 414, in on_epoch_end
    callback.on_epoch_end(epoch, logs)
TypeError: on_epoch_end() takes from 1 to 2 positional arguments but 3 were given